store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/retriever/impl/AccumuloElementsRetriever.java

```diff
@@ -19,20 +19,27 @@
 import uk.gov.gchq.gaffer.accumulostore.AccumuloStore;
 import uk.gov.gchq.gaffer.accumulostore.key.exception.IteratorSettingException;
 import uk.gov.gchq.gaffer.data.element.id.EdgeId;
-import uk.gov.gchq.gaffer.operation.impl.get.GetElements;
+import uk.gov.gchq.gaffer.operation.Operation;
 import uk.gov.gchq.gaffer.store.StoreException;
 import uk.gov.gchq.gaffer.user.User;
 
 import java.util.stream.StreamSupport;
 
-public class AccumuloElementsRetriever extends AccumuloSingleIDRetriever<GetElements> {
+public class AccumuloElementsRetriever extends AccumuloSingleIDRetriever<Operation> {
     public AccumuloElementsRetriever(final AccumuloStore store,
-                                     final GetElements operation,
+                                     final Operation operation,
                                      final User user)
             throws IteratorSettingException, StoreException {
+        Iterable<?> input;
+        if (operation.getInput() instanceof Iterable) {
+            input = Iterable.class.cast(operation.input());
```

Comment:
This is check is done by the handler. This is kinda duplicate checking.
either the Handler gives this pre-casted object and pass it down or don't bother with the check and just cast it here.

(kinda feels unsafe like Class Cast exceptions will be thrown, but the what's difference with that and throwing a `new IllegalArgumentException("The input is not of type");`)

Reply:
Have just cast now and have removed the if-or-error block